README.md

```diff
@@ -109,10 +126,10 @@ example.go:5:10: !(v1 != v2)
 
 It automatically inserts `Report("$$")` into the specified pattern.
 
-## How does it work?
+## How it works
```

Comment:
why?

Reply:
[My linter](https://github.com/DavidAnson/vscode-markdownlint) told me to do so. Namely, it is [MD026](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md026---trailing-punctuation-in-heading) check. My motivation is that this is not an FAQ section, so you should answer questions, not to ask them. Also, [Betteridge's law of headlines](https://en.wikipedia.org/wiki/Betteridge%27s_law_of_headlines) is **a bit** related as an example of why you should avoid question mark in headlines.

And as the last piece of motivation let's use crowd intelligence:
+ https://github.com/search?q=%22How+it+works%22 (2 323 matches)
+ https://github.com/search?q=%22How+does+it+work%22 (103 matches)

Reply:
lol, interesting